### DISCUSSION TITLE
score bug

### DISCUSSION THREAD

**Discussion by moha-b** [moha-b/Flappy-Bird#21] [Q&A]
When you press the start button, quickly **after losing the first** time the points will increase by themselves

**Comment by narmuni123** [comment]
Solved the issue
if(!birdIsDead() && gameHasStarted == true){
          setState(() {
            if (score == topScore) {
              topScore++;
            }
            score++;
          });
        }

Try this

**Comment by moha-b** [comment]
I already try it but it's not working this is the code from the game
```dart
        setState(() { 
           if (score == topScore) { 
             topScore++; 
           } 
           score++; 
         });
```
